gguf-py/gguf/gguf.py

```diff
@@ -723,6 +754,84 @@ def add_pad_token_id(self, id: int):
         self.add_uint32(KEY_TOKENIZER_PAD_ID, id)
 
 
+class SpecialVocab:
+    load_merges: bool = False
+    merges: List[str] = []
+    special_token_types: Tuple[str, ...] = tuple(('bos', 'eos', 'unk', 'sep', 'pad'))
+    special_token_ids: Dict[str, int] = {}
+
+    def __init__(self, path: Path, load_merges: bool = False, special_token_types: Optional[Tuple[str, ...]] = None):
```

Comment:
Requiring a pathlib.Path in the API is not user-friendly. Generalizing it to os.PathLike[str] would be better.

Reply:
Makes sense to me.